Solon_Burleson/DLists.py

```diff
@@ -0,0 +1,66 @@
+class Node:
+    def __init__(self, value):
+        self.value = value
+        self.next = None
+        self.prev = None
+
+class Dlist:
+    def __init__(self,value):
+        node = Node(value)
+        self.head = node
+
+    def addNode(self, value):
+        node = Node(value)
+        runner = self.head
+        while runner.next != None:
+            runner = runner.next
+        runner.next = node
+        node.prev = runner
+
+    def removeNode(self, value):
+        runner = self.head
+        if runner.value == value:
+            runner = runner.next
+            self.head = runner
+        prevrunner = runner
+        while runner.next != None:
+            if runner.value == value:
+                runner = runner.next
+                prevrunner.next = runner
+                runner.prev = prevrunner                
```

Comment:
if runner != None then runner.prev = prevrunner //to remove the last second node